### PR TITLE
Safety off

### PR DESCRIPTION
**Pull request by 5yler**
### Background

- Bump up high force and torque limit for collision detection on Franka
- 
### What's new
- ✅ [New feature description. Only a single new major feature is allowed per PR.]
- ❌ New feature is accompanied by new test: [name and description of new test].

### Related work
- ✅❌ This PR needs [link]
- ✅❌ [link] needs this PR

### Tests
1. ✅ Tested on simulated robot: Tested value gets set based on env var
2. ✅ Tested on physical robot: TLE on ranch
### Quality
3. ✅ New code is written in pure C++17 to the best of my knowledge.
4. ✅ New code follows established C++17 best practices ([C++ Core Guidelines](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines)).
5. ✅ New code passed `clang-format` and `cpplint` at least, if not all of our static code analysers.
6. ✅ I have included Doxygen-style documentation in the new C++ code.

### Note to reviewers
Please verify that all sections are accurately filled out and that all 6 numbered entries above are present and ticked/checked off, with their requirements met, if applicable.

### CODE REVIEW

File: src/franka_plan_runner.cc [hunk 1]
```diff
@@ -115,14 +126,13 @@ void FrankaPlanRunner::SetCollisionBehaviorSafetyOn(franka::Robot& robot) {
...
-                             {{40.0, 40.0, 36.0, 36.0, 32.0, 28.0, 24.0}},
-                             {{40.0, 40.0, 36.0, 36.0, 32.0, 28.0, 24.0}},
-                             {{40.0, 40.0, 40.0, 50.0, 50.0, 50.0}},
-                             {{40.0, 40.0, 40.0, 50.0, 50.0, 50.0}},
-                             {{40.0, 40.0, 40.0, 50.0, 50.0, 50.0}},
-                             {{40.0, 40.0, 40.0, 50.0, 50.0, 50.0}});
+
+  // Changes the collision behavior. Forces or torques above the upper
+  // threshold are registered as collision and cause the robot to stop moving.
+  robot.setCollisionBehavior(upper_torque_threshold_, upper_torque_threshold_,
```
Comment (gavincangan) [inline src/franka_plan_runner.cc:132]: Could you add a comment about why we need to pass this four times?

File: src/franka_plan_runner.cc [hunk 2]
```diff
@@ -132,15 +142,12 @@ void FrankaPlanRunner::SetCollisionBehaviorSafetyOff(franka::Robot& robot) {
...
-      {{100.0, 100.0, 100.0, 100.0, 100.0, 100.0, 100.0}},
-      {{100.0, 100.0, 100.0, 100.0, 100.0, 100.0, 100.0}},
-      {{100.0, 100.0, 100.0, 100.0, 100.0, 100.0, 100.0}},
-      {{100.0, 100.0, 100.0, 100.0, 100.0, 100.0}},
-      {{100.0, 100.0, 100.0, 100.0, 100.0, 100.0}},
-      {{100.0, 100.0, 100.0, 100.0, 100.0, 100.0}},
-      {{100.0, 100.0, 100.0, 100.0, 100.0, 100.0}});
+  // Changes the collision behavior. Forces or torques above the upper threshold
+  // are registered as collision and cause the robot to stop moving.
+  robot.setCollisionBehavior(kHighTorqueThreshold, kHighTorqueThreshold,
```
Comment (gavincangan) [inline src/franka_plan_runner.cc:147]: Could you add a comment about why we need to pass this four times?

**Review by gavincangan — APPROVED** [review]
lgtm style and platform

File: src/franka_plan_runner.cc (hunk 1 above)
Comment (5yler) [reply src/franka_plan_runner.cc:150]: Yeah.

**Pull request merged**